utils/bmx7-dnsupdate/Makefile

```diff
@@ -0,0 +1,27 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bmx7-dnsupdate
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/bmx7-dnsupdate
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=bmx7-dnsupdate
+  MAINTAINER:=Paul Spooren <spooren@informatik.uni-leipzig.de>
+  DEPENDS:=+bmx7 +bmx7-json +dnsmasq +inotifywait
```

Comment:
what about users of dnsmasq-dhcpv6 and dnsmasq-full? i reckon it doesn't make sense to depend on dnsmasq.

Reply:
Removed